cypress/support/step_definition/login_management/invalidCredentials.spec.js

```diff
@@ -2,34 +2,34 @@ import Login_Management from '../../pages/login_management/pageLoginManagement'
 
 const loginManagement = new Login_Management()
 
-Then('I will enter invalid email', () => {
+Then('I enter invalid email', () => {
   loginManagement.enterInvalidEmailOrUsername(true)
 })
 
-Then('I will enter invalid username', () => {
+Then('I enter invalid username', () => {
   loginManagement.enterInvalidEmailOrUsername(false)
 })
 
-Then('I will enter invalid password', () => {
+Then('I enter invalid password', () => {
   loginManagement.enterInvalidPassword()
 })
 
-Then('I will see error message', () => {
+Then('I see error message', () => {
   loginManagement.errorMessageDisplay()
```

Comment:
method name should be here - >displayErrorMessage(), considering verb+noun rule in method names